### ISSUE TITLE
Command for a team to win

### ISSUE THREAD

**Issue by propain101** [alkarinv/BattleArena#204]
It would be nice if there was a command like "/arena Winner team 1" or somthing like that, it would then be posible to use redstone and command Blocks to decide wich teams win :)

Regards
Propain

**Comment by xieno** [comment]
Currently "/arena winner username" should provide that same effect.

**Comment by propain101** [comment]
Yes i am aware of that command, but if your using redstone, or commands Blocks it could register wrongly and think that the other team won
